solr/core/src/java/org/apache/solr/servlet/HttpSolrCall.java

```diff
@@ -934,13 +934,7 @@ protected void logAndFlushAdminRequest(SolrQueryResponse solrResp) throws IOExce
    * returns the default query response writer Note: This method must not return null
    */
   protected QueryResponseWriter getResponseWriter() {
-    String wt = solrReq.getParams().get(CommonParams.WT);
```

Comment:
I love this werid logic being removed!

Reply:
it has moved:
> Moved HttpSolrCall.getResponseWriter to SolrQueryRequest

It wasn't specific to HTTP; EmbeddedSolrServer needs to call the same logic.